### PR TITLE
Add tests for some functionality that applies to many commands

### PR DESCRIPTION
**Pull request by KSchopmeyer**
Note that this also corrects two errors found during this testing.

See commit for details

### CODE REVIEW

File: pywbemtools/pywbemcli/_cmd_instance.py [hunk 1]
```diff
@@ -833,15 +835,18 @@ def cmd_instance_modify(context, instancename, options):
 
     If successful, this operation returns nothing.
     """
+    context.spinner_stop()
```
Comment (andy-maier) [inline pywbemtools/pywbemcli/_cmd_instance.py]: Is there a reason for this spinner_stop() call or is this a remainder of a debug session?

Comment (KSchopmeyer) [reply pywbemtools/pywbemcli/_cmd_instance.py]: Arrgh hangover from debug session.  Actually should be applied for every case of the if verbose pattern to stop spinner before outputting the message.

Comment (KSchopmeyer) [reply pywbemtools/pywbemcli/_cmd_instance.py]: Note that this did point out an issue where I was not consisently turning off spinner before those verbose outputs.  **Fixed**

**Comment by KSchopmeyer** [comment]
Issues fixed, passes tests.

**Pull request merged**